compiler/test/dotty/tools/dotc/SettingsTests.scala

```diff
@@ -29,13 +29,12 @@ class SettingsTests {
     assertTrue(Files.exists(out))
   }
 
-  @Test def t8124: Unit = {
-    val source = Paths.get("tests/pos/Foo.scala").normalize
+  @Test def t8124: Unit =
+    val source    = Paths.get("tests/pos/Foo.scala").normalize
     val outputDir = Paths.get("out/testSettings").normalize
-    if (Files.notExists(outputDir)) Files.createDirectory(outputDir)
-    val options = Array("-encoding", "-d", outputDir.toString, source.toString)
-    val reporter = Main.process(options)
+    if Files.notExists(outputDir)
+      Files.createDirectory(outputDir)
+    val options  = Array("-encoding", "-d", outputDir.toString, source.toString)
```

Comment:
The argument is missing for `-encoding`. Or remove it.

Reply:
The ticket is about the missing arg.